sdk/nodejs/docker.ts

```diff
@@ -132,9 +132,10 @@ export function buildAndPushImageAsync(
     pathOrBuild: pulumi.Input<string | DockerBuild>,
     repositoryUrl: pulumi.Input<string>,
     logResource: pulumi.Resource,
+    skipPush: pulumi.Input<boolean>,
```

Comment:
This should just be a `boolean`. Though an `Input<boolean>` would allow the value of this argument to depend on the outputs of other resources, it would also require some refactoring in order to work properly: as written, the push will be skipped any time this argument is an `Output<boolean>`, even if that output resolves to `false`.

Reply:
Got it! and fixed.